index.html

```diff
@@ -79,6 +79,191 @@ <h2>LET'S CONNECT</h2>
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <div class="cta-button">See Project</div>
+        </div>
+      </div>
+      
+      <div class="card" id="card3">
+        <div class="card-img">
+          <img src="images/pic3.png" alt="Facebook 360 sneakpeak">
+        </div>
+        <div class="card-text">
+          <h2>Facebook 360</h2>
+          <ul class="categories">
+            <li class="canopy">Facebook</li>
+            <li>•</li>
+            <li>Full Stack Dev</li>
+            <li>•</li>
+            <li>2015</li>
+          </ul>
+          <p>A daily selection of privately personalized reads; 
+            no accounts or sign-ups required</p>
+          <ul class="keyword">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <div class="cta-button">See Project</div>
+        </div>
+      </div>
+
+      <div class="card" id="card4">
+        <div class="card-img">
+          <img src="images/pic4.png" alt="Uber navigation sneakpeak">
+        </div>
+        <div class="card-text">
+          <h2>Uber Navigation</h2>
+          <ul class="categories">
+            <li class="canopy">Uber</li>
+            <li>•</li>
+            <li>Lead developer</li>
+            <li>•</li>
+            <li>2018</li>
+          </ul>
+          <p>A daily selection of privately personalized reads; 
+            no accounts or sign-ups required</p>
+          <ul class="keyword">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <div class="cta-button">See Project</div>
+        </div>
+      </div>
+
+    </section> <!-- End of PORTFOLIO -->
+
+    <section class="about" id="About">
+    <div class="about-elements">
+    <div class="about1">
+      <div class="content">
+        <h2>About <br>Myself</h2>
+        <p>
+          I'm a software developer! I can help you 
+          build a product, feature or website Look 
+          through some of my work and 
+          experience! If you like what you see and 
+          have a project you need coded, don't 
+          hesitate to contact me.
+        </p>
+      </div>
+      <div class="socials">
+        <h2>LET'S CONNECT</h2>
+        <ul class="icon-bar">
+          <li><a href="#">
+            <img src="icons/twitter.png" alt="Twitter logo">
+          </a></li>
+          <li><a href="#">
+            <img src="icons/linkedin.png" alt="Linkedin logo">
+          </a></li>
+          <li><a href="#">
+            <img src="icons/facebook.png" alt="Facebook logo">
+          </a></li>
+          <li><a href="#">
+            <img src="icons/github.png" alt="Github logo">
+          </a></li>
+          <li><a href="#">
+            <img src="icons/instagram.png" alt="Instagram logo">
+          </a></li>
+        </ul>
+        <div class="cta-button">Get my resume</div>
+      </div>
```

Comment:
Kindly change this div to a button or a link (a tag). Keep coding 💪

Reply:
done